femtools/Diagnostic_variables.F90

```diff
@@ -31,7 +31,7 @@ module diagnostic_variables
   !!< A module to calculate and output diagnostics. This replaces the .s file.
   use fldebug 
   use global_parameters, only:FIELD_NAME_LEN,OPTION_PATH_LEN, &
-& PYTHON_FUNC_LEN, int_16, integer_size, real_size
+& PYTHON_FUNC_LEN, integer_size, real_size
```

Comment:
Does this one need a `use iso_c_binding` as well? I'm guessing it currently inherits it from somewhere...